back/engines/commercial/multi_tenancy/lib/tasks/core/create_tenant.rake

```diff
@@ -354,8 +354,8 @@ namespace :cl2_back do # rubocop:disable Metrics/BlockLength
           allowed: true
         },
         homepage_builder: {
-          enabled: false,
-          allowed: false
+          enabled: true,
+          allowed: true
```

Comment:
This enables the feature for epic deployments and the e2e tests. I will make a separate PR to enable it by default and prepare rake tasks to specify this for the pricing plans, so that the feature is also enabled for new tenants. I'm not doing it here, because we first need to:
- Run the script to migrate the craftjs_json of the homepages
- Fix possible issues manually with the external template platforms

Reply:
And also regenerate the external templates

Reply:
Oh hadn't realised this config was not used when creating tenants on staging/production